tests/unit/services/sl-translate-test.js

```diff
@@ -28,139 +28,67 @@ test( 'dictionary property defaults to null', function( assert ) {
+    const testProperty = Ember.Object.create({
+        parameter: ''
+    });
 
-    // Empty parameter
-
-    let assertionThrown = false;
+    const callSetDictionary = () => TS.setDictionary( testProperty.parameter );
 
-    try {
-        TS.setDictionary();
-    } catch( error ) {
-        assertionThrown = true;
-    }
+    testProperty.set( 'parameter', [] );
 
-    assert.ok(
-        assertionThrown,
-        'Parameter was empty'
+    assert.throws(
+        callSetDictionary,
+        'Parameter was an array'
     );
 
-    // Null parameter
-
-    assertionThrown = false;
+    testProperty.set( 'parameter', null );
 
-    try {
-        TS.setDictionary( null );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
+    assert.throws(
+        callSetDictionary,
         'Parameter was null'
     );
 
-    // Number parameter
-
-    assertionThrown = false;
-
-    try {
-        TS.setDictionary( 4 );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
-        'Parameter was a Number'
-    );
-
-    // Array Parameter
+    testProperty.set( 'parameter', 123 );
 
-    assertionThrown = false;
-
-    try {
-        TS.setDictionary( [] );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
-        'Parameter was an Array'
+    assert.throws(
+        callSetDictionary,
+        'Parameter was a number'
     );
 
-    // Function
-
-    assertionThrown = false;
+    testProperty.set( 'parameter', function() {} );
 
-    try {
-        TS.setDictionary( function() {} );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
-        'Parameter was a Function'
+    assert.throws(
+        callSetDictionary,
+        'Parameter was a function'
     );
 
-    // String Parameter
-
-    assertionThrown = false;
-
-    try {
-        TS.setDictionary( 'test' );
-    } catch( error ) {
-        assertionThrown = true;
-    }
+    testProperty.set( 'parameter', 'testString' );
 
-    assert.ok(
-        assertionThrown,
-        'Parameter was a String'
+    assert.throws(
+        callSetDictionary,
+        'Parameter was a string'
     );
 
-    // Boolean Parameter
-
-    assertionThrown = false;
+    testProperty.set( 'parameter', false );
 
-    try {
-        TS.setDictionary( false );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
-        'Parameter was a Boolean'
+    assert.throws(
+        callSetDictionary,
+        'Parameter was false'
     );
 
-    // Object Parameter
+    testProperty.set( 'parameter', {} );
 
-    assertionThrown = false;
-
-    try {
-        TS.setDictionary( {} );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        !assertionThrown,
-        'Parameter was an Object'
+    assert.equal(
```

Comment:
The setDictionary function returns undefined, therefore we cannot use assert.ok, that is the reason assert.equal tests are included.